modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -19,6 +19,58 @@ Replacement syntax for deprecated and removed features are also indicated.
 [[cypher-deprecations-additions-removals-5.17]]
 == Neo4j 5.17
 
+=== Updated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role=noheader]
+----
+CREATE [index_type] INDEX [index_name] IF NOT EXISTS FOR ...
+----
+|
+When attempting to create an index using `IF NOT EXISTS` with either the same name or same index type and schema, or both,
+as an existing index the command now returns a notification showing the existing index which blocks the creation.
```

Comment:
Please don't break text in-between sentences. Either break it after full stops, or not at all. There's multiple occurrences in this file.

Reply:
I did break them while writing to be able to see the sentences since they were so long (as I don't have soft wrapping so the lines just continue out of sight and requires scrolling to see the end) but I can definitely go back and fix that 👍